presto-native-execution/README.md

```diff
@@ -96,17 +85,90 @@ checks.  When we need one it can be added.
 
 ## Development Env.
 
-### Setting up on macOs
-
-See `scripts/setup-macos.sh`
-
-After running the setup script add the cmake-format bin to your $PATH, maybe
-something like this in your ~/.profile:
+### Basic setting up on MacOs
 
+* Clone the whole Presto repository.
+* Run `presto/presto-native-execution/scripts/setup-macos.sh`.
+* Add the cmake-format bin to your $PATH, maybe something like this
+  in your ~/.profile:
 ```
 export PATH=$HOME/bin:$HOME/Library/Python/3.7/bin:$PATH
 ```
+* It is recommended to use some sort of IDE on Mac. CLion is an option.
 
 ### Setting up on Linux (CentOs 8 or later)
 
-See `scripts/setup-linux.sh`
+* Clone the whole Presto repository.
+* Run `presto/presto-native-execution/scripts/setup-linux.sh`
+
+### Setting up local Presto environment on MacOs (with optional IntelliJ and CLion) 
+Clone the whole Presto repository.
+
+Close IntelliJ and CLion
+
+In local Presto repo
+* `git fetch upstream`
+* `git co upstream/master`
+* `mvn clean install -DskipTests -T1C -pl -presto-docs`
```

Comment:
What does "-pl -presto-docs" do? Should we exclude presto-docs from building here?

Reply:
Not sure what -pl does, but the other one excludes presto docs - we don't need them.
Actually, likely many of the modules we don't need, but the docs would build at some point, so I got them excluded. :)

Reply:
I see that '-' can be used to specify a module to exclude: https://maven.apache.org/guides/mini/guide-multiple-modules-4.html#inclusion-and-exclusion

I'm used to '!', hence, wasn't sure what this command is for.

The documentation says both ! and - can be used.